lib/CurrentUser.php

```diff
@@ -85,7 +85,7 @@ public function getUID(): ?string {
 		if ($this->sessionUser === false) {
 			$user = $this->userSession->getUser();
 			if ($user instanceof IUser) {
-				$this->sessionUser = (string) $user->getUID();
+				$this->sessionUser = (string) $user->getDisplayName();
```

Comment:
the function is called `getUID`
What is actually missing is a translation of the detail in the activity afterwards when it is rendered?
We shouldn't write the display name into the activity.
But then again this works fine here.

Reply:
![Bildschirmfoto von 2022-02-23 16-35-16](https://user-images.githubusercontent.com/213943/155352396-fbf28c6b-4ddb-4c5c-8802-22955af78e44.png)

Reply:
User handling and getting the display name is actually here:
https://github.com/nextcloud/files_downloadactivity/blob/2482f99fa20602061e985e40f965427f3f7cbd67/lib/Activity/Provider.php#L230-L253

Reply:
The bug actually is from the trackdownload app which is also adding the IP address (not as a second parameter) to the userId: 
https://github.com/TimoKunze/files_trackdownloads/blob/master/lib/Activity/Listener.php#L113